packages/@react-types/image/src/index.d.ts

```diff
@@ -14,29 +14,17 @@ import {DOMProps, StyleProps} from '@react-types/shared';
 
 export interface ImageProps {
   /**
-   * Whether the Image should be displayed with a loaded style.
+   * The URL of the image.
    */
-  loaded?: boolean,
+  src: string,
   /**
-   * Whether the Image should be displayed with a placeholder style.
+   * Text description of the image.
    */
-  isPlaceholder?: boolean,
+  alt: string,
   /**
    * Sets the Image [object-fit](https://developer.mozilla.org/en-US/docs/Web/CSS/object-fit) style.
    */
   objectFit?: any, // move to styleProps for images and type better
-  /**
-   * The URL of the image.
-   */
-  src?: string,
-  /**
-   * Whether the Image should be decorative and not have an `alt` property.
-   */
-  decorative?: boolean,
```

Comment:
Just curiosity, why no more decorative?

Reply:
All it does is set `alt` to empty string so we may as well just let people do that directly and make it required... or "mando"

Reply:
`alt` is now a required prop, and `alt=""` is equivalent.